config/application.rb

```diff
@@ -66,6 +66,18 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :delayed_job
 
+    config.conference = {
+      events_per_page: 3,
+      default_logo:    'snapcon_logo.png',
```

Comment:
These things are good, but I think we should make them be environment variables, with a default value. `DEFAULT_LOGO_FILENAME || ...` type deal.